ops/testing.py

```diff
@@ -37,29 +37,22 @@
 
 import importlib.metadata
 
+from . import charm, framework, model, pebble, storage
 from ._private.harness import (
     ActionFailed,
     ActionOutput,
     AppUnitOrName,
-    CharmBase,
-    CharmMeta,
     CharmType,
-    Container,
     ExecArgs,
     ExecHandler,
-    ExecProcess,
     ExecResult,
     Harness,
     ReadableBuffer,
-    RelationNotFoundError,
-    RelationRole,
     YAMLStringOrFile,
-    charm,
-    framework,
-    model,
-    pebble,
-    storage,
 )
+from .charm import CharmBase, CharmMeta, RelationRole
```

Comment:
What if someone imports `ops.testing.charm` today 🤔 
Let's validate that none actually uses the 5 disappearing re-exports.

Reply:
No, they're still imported -- see line 40. Does that help?

Reply:
Oh sorry. I didn't realise that `ops._private.harness.charm is ops.charm`...